python/ray/tests/test_worker_capping.py

```diff
@@ -142,43 +140,6 @@ def foo():
     assert len(not_ready) == 0
 
 
-def test_exponential_wait(shutdown_only):
```

Comment:
I feel this test may not be that useful: we try to test the exponential wait of worker capping but in the integration test, we can only measure the end-to-end wall clock time which involves many things out of control (e.g. worker process launch time, PushTask rpc time, etc). If we have the assertion too tight then this test is flaky but if it's too loose we may not catch bugs.

I think the existing unit test is probably enough to test exponential wait and it is deterministic. 

WDYT? @wuisawesome @ericl

Reply:
Hmm, I think an integration test here is still valuable. Perhaps we can have some very very loose constraints? E.g., just check that things increased at all?

Reply:
Yea, we can make the assertion loose enough to pass the test. I'm just not sure what signal a passing test can give us: the increased wait time that we can measure can be caused by other factors not the exponential_wait code.

Also it's hard to test if the wait is actually EXPONENTIAL increase or some other way of increase. If you are saying we don't care here how the wait time is increased but as long as it's increased, it's good, then we can probably test that.

Reply:
Well, I think testing something like the first wait is <1s, and the last wait is >8s would be pretty meaningful. Without such a test it would be easy for the wait to be broken when running e2e.

Reply:
Ok, I think I find a way to make the test not flaky (the lower bound is guaranteed by exponential_wait, but the upper bound depends on other factors). Let me know if that looks good.